stdlib/influxdata/influxdb/to.go

```diff
@@ -148,48 +130,37 @@ func (t *ToTransformation) Process(id execute.DatasetID, tbl flux.Table) error {
 			}
 		}
 
-		addTagsFromTable(t.spec, tbl, excludeColumns)
+		t.addTagsFromTable(chunk.Cols(), excludeColumns)
 	}
-	return writeTableToAPI(t.ctx, t, tbl)
-}
 
-// UpdateWatermark updates the watermark for the transformation for the `to` flux function.
-func (t *ToTransformation) UpdateWatermark(id execute.DatasetID, pt execute.Time) error {
-	return t.d.UpdateWatermark(pt)
-}
+	if err := t.writeTable(chunk); err != nil {
+		return err
+	}
 
-// UpdateProcessingTime updates the processing time for the transformation for the `to` flux function.
-func (t *ToTransformation) UpdateProcessingTime(id execute.DatasetID, pt execute.Time) error {
-	return t.d.UpdateProcessingTime(pt)
+	filtered := t.filterNulls(chunk, mem)
+	return d.Process(filtered)
```

Comment:
Could you add some comments here about the null filtering? My understanding of what you're doing here is:
- The call to `writeTable` above will skip over any points that have a null timestamp
- Because the semantics of `to()` is that we output the tables that are written, we then need to filter rows with null timestamps from the input.

Are there tests for the null filtering?

Reply:
Yes, there's a test for this. I didn't know about this behavior before I saw the failed test.

Reply:
I'll also add a comment.

Reply:
I updated the doc string for `filterNulls` to explain what is going on.